tmt/steps/provision/__init__.py

```diff
@@ -292,7 +292,11 @@ def ansible(self, playbook: str, extra_args: Optional[str] = None) -> None:
 
         raise NotImplementedError()
 
-    def execute(self, command: Union[str, List[str]], **kwargs: Any) -> tmt.utils.CommandOutput:
+    def execute(self,
+                command: Union[str,
+                               List[str]],
```

Comment:
This can fit on a single line. Similar below.

Reply:
Fixed, I hope, in 45e9bd5c654d8c31645fc2e630f2d4ebcfdbd28d.